src/mlpack/core/cv/metrics/r2_score_impl.hpp

```diff
@@ -45,7 +46,14 @@ double R2Score::Evaluate(MLAlgorithm& model,
   // Handling undefined R2 Score when both denominator and numerator is 0.0.
   if (residualSumSquared == 0.0)
     return totalSumSquared ? 1.0 : DBL_MIN;
-
+  // Returning adjusted R-squared
```

Comment:
```suggestion
  // Returning adjusted R-squared.
```

Missing stop at the end.